doc/user/flowspec.rst

```diff
@@ -0,0 +1,330 @@
+is received by *zebra*. Two filtering contexts will be created or appended (`Linux
+NetFilter ipset` and `Linux netfilter iptable` entry). If the traffic has to be dropped,
+then those filtering contexts will be enough. Otherwise, the traffic will be redirected
+to an other routing table, with in that routing table, a route entry to redirect the
+traffic to the wished destination.
+
+Configuration guide
+-------------------
+
+In order to configure an IPv4 flowspec engine, use the following configuration. As of
+today, it is only possible to configure Flowspec on default VRF.
+
+.. code-block:: frr
+
+   router bgp <AS>
+     neighbor <A.B.C.D> remote-as <remoteAS>
+     address-family ipv4 flowspec
+      neighbor <A.B.C.D> activate
+    exit
+   exit
+
+You can see Flowspec entries, by using one of the following show commands:
+
+.. index:: show bgp ipv4 flowspec [detail | A.B.C.D]
+.. clicmd:: show bgp ipv4 flowspec [detail | A.B.C.D]
+
+
+Flowspec Per Interface
+~~~~~~~~~~~~~~~~~~~~~~
+
+One nice feature to use is the ability to apply flowspec to a specific interface,
+instead of applying it to the whole machine. Despite the following IETF draft
+[Draft IETF IDR Flowspec Interface Set]_ is not implemented, it is possible to
+manually limit Flowspec application to some incoming interfaces. Actually, not
+using it can result to some unexpected behaviour like accounting twice the traffic,
+or slow down the traffic (filtering costs). To limit flowspec to one specific
+interface, use the following command, under flowspec address-family node.
+
+.. index:: [no] local-install <IFNAME | any>
+.. clicmd:: [no] local-install <IFNAME | any>
+
+By default, Flowspec is activated on all interfaces. Installing it to a named interface
+will result in allowing only this interface. Reversely, enabling any interface will flush
+all previously configured interfaces.
+
+Flowspec redirect VRF
+~~~~~~~~~~~~~~~~~~~~~
+
+An other nice feature to configure is the ability to redirect traffic to a separate VRF.
+This feature does not go against the ability to configure Flowspec only on default VRF.
+Actually, when you receive incoming BGP flowspec entries on that default VRF, you can
+redirect traffic to an other VRF.
+
+As remind, BGP flowspec entries have a BGP extended community that contains a Route Target.
+Finding out a local VRF based on Route Target consists in the following:
+
+- A configuration of each VRF must be done, with its Route Target set
+
+Each VRF is being configured within a BGP VRF instance with its own Route Target list.
+Route Target accepted format matches the following: A.B.C.D:U16, or U16:U32, U32:U16.
+
+- The first VRF with the matching Route Target will be selected to route traffic to. Use
+the following command under ipv4 unicast address-family node
+
+.. index:: [no] rt redirect import [Route Target List]
+.. clicmd:: [no] rt redirect import [Route Target List]
```

Comment:
Please use the syntax used in the codebase.

Reply:
ok